kak-lsp.service

```diff
@@ -0,0 +1,10 @@
+[Unit]
+Description=Kak LSP daemon
+
+[Service]
+ExecStart=/home/lars/.local/bin/kak-lsp
```

Comment:
Is it possible to use `~` or `$HOME` or some equivalent here? If not, we probably should leave a note in README about extra step of editing this file.